charts/kubernikus/templates/ingress.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   name: kubernikus-api
   annotations:
     vice-president: "true"
+    prometheus.io/scrape: "true"
+    prometheus.io/probe: "true"
```

Comment:
need to add the `prometheus.io/probe: "true"` to consider the ingress for blackbox testing.
note to self: additional `prometheus.io/scrape: "true"` shouldn't be necessary

Reply:
To further clarify:
These annotations are telling prometheus to do two *different* things.
`prometheus.io/probe: true` is telling prometheus to give this ingress to the blackbox exporter to test for http metrics (and return the result to prometheus).
`prometheus.io/scrape: true` is telling prometheus to scrape the ingresses targets itself for built-in prometheus metrics. Which is usually already done on pod level.

Reply:
agreed. that's why I left the note.